Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations_test.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from QRadarGetOffenseCorrelations import get_query
+
+
+@pytest.mark.parametrize('bool_val', ["False"])
+def test_get_query_cre_name_null_false(bool_val):
+    query_from_false = get_query(bool_val)
```

Comment:
I think in tests you better mock the demisto.args() - pass is_cre_name_null=True and mock demisto.executeCommand and validate that the passed query contains `"\"CRE NAME\" <> NULL"` or you can compare the whole query

Reply:
How would that improve the test from your perspective?

Reply:
it will make it more end to end test.